packages/frontend/src/components/accounts/ledger/SetupLedger.js

```diff
@@ -93,7 +103,7 @@ const SetupLedger = (props) => {
                             return;
                         }
 
-                        await dispatch(createNewAccount({ accountId, fundingOptions, recoveryMethod: 'ledger', publicKey, recaptchaToken })).unwrap();
+                        await dispatch(createNewAccount({ accountId, fundingOptions, recoveryMethod: 'ledger', publicKey, recaptchaToken, path: customLedgerHdPath })).unwrap();
```

Comment:
I don't see `path` being used in `createNewAccount`? Also, since createNewAccount is not ledger-specific, can we name it `ledgerHDPath` or similar?

Reply:
Thanks for spotting this. It's residual code from a refactor